config.py

```diff
@@ -44,6 +44,11 @@ class Config(object):
     PARTY_PASSWORD = os.getenv('PARTY_PASSWORD')
     PARTY_AUTH = (PARTY_USERNAME, PARTY_PASSWORD)
 
+    RM_REPORT_SERVICE_HOST = os.getenv('RM_REPORT_SERVICE_HOST', 'localhost')
+    RM_REPORT_SERVICE_PORT = os.getenv('RM_REPORT_SERVICE_PORT', 8084)
+    RM_REPORT_SERVICE_PROTOCOL = os.getenv('RM_REPORT_SERVICE_PROTOCOL', 'http')
+    RM_REPORT_SERVICE = f'{RM_REPORT_SERVICE_PROTOCOL}://{RM_REPORT_SERVICE_HOST}:{RM_REPORT_SERVICE_PORT}/'
```

Comment:
Should be using the new NAME_URL service form with defaults (only?) in the DevelopmentConfig.

Do we also need separate auth details for the report service?

Reply:
I presume to see the env vars to be condensed